scripts/gceworker.sh

```diff
@@ -74,6 +74,19 @@ case "${cmd}" in
       read COCKROACH_DEV_LICENSE
     fi
 
+    DEFAULT_SSH_KEY_PATH="$HOME/.ssh/id_rsa"
```

Comment:
I suggest always using curly braces when using variables in shell scripts. This helps avoid problems. This also matches the usage in the rest of the scripts. If you make this change, then please change it for all your code changes. 

```suggestion
    DEFAULT_SSH_KEY_PATH="${HOME}/.ssh/id_rsa"
```